backend/resources/hacknight.py

```diff
@@ -54,3 +54,38 @@ def get(self, id):
         return {'hacknights': hacknight_schema.dump(Hacknight.query.get_or_404(
             id))
         }, HTTPStatus.OK
+
+    @jwt_required
+    def patch(self, id):
+        hacknight = Hacknight.query.get_or_404(id)
+        participants = [
+            participant.id for participant in hacknight.participants
+        ]
+
+        json_data = request.get_json(force=True)
+        ids_schema = Schema.from_dict(
+            {"participants_ids": fields.List(fields.Int())}
+        )
+        try:
+            data = ids_schema().load(json_data)
+        except ValidationError as err:
+            return (err.messages), HTTPStatus.BAD_REQUEST
+
+        new_participants = [
```

Comment:
As I understand that this endpoint should be able to change the list of participants for particular hacknight.

I see that you implemented only adding new participants? How the plan for removing a participant from hacknight looks like?

Reply:
Title of these ticket is "Create add participant to hacknight endpoint", and I believe that delete participant from hacknight, is great topic for next one

Reply:
Chenged to post on /hacknights/<id>/participants @ngiersz going to do delete method